tests/unit/utils/test_bundle_utils.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
move this with existing imports

Reply:
It has to be at the beginning of the file (more exactly: https://docs.python.org/2/reference/simple_stmts.html#future)
I add this line in order to run the unit test in pycharm because of that issue: https://stackoverflow.com/questions/38569992/pycharm-import-runtimewarning-after-updating-to-2016-2